cmd/security-secrets-setup/start_vault.sh

```diff
@@ -24,21 +24,16 @@ VAULT_TLS_PATH=${VAULT_TLS_PATH:-/tmp/edgex/secrets/edgex-vault}
 DEFAULT_VAULT_LOCAL_CONFIG='
 listener "tcp" { 
               address = "edgex-vault:8200" 
-              tls_disable = "0" 
+              tls_disable = "1" 
```

Comment:
Should we also get rid of the `VAULT_TLS_PATH` logic in this script, and in particular the `SENTILNEL_FILE` loop?

Reply:
Tony @tonyespy, at this stage, we still need that since the `secrets-setup` has not been removed yet and thus docker consul health checkers still checks that I guess.  Removing secrets-setup is another big step on snap as well.